app/components/review_component/review_component.scss

```diff
@@ -100,3 +100,10 @@
 .action-required-tag {
   @extend .notification-tag;
 }
+
+.mutliline {
```

Comment:
mutli => multi

Reply:
ah mutli is the dog from wacky races! @david-mears-dfe